cli/lib/kontena/cli/stacks/stacks_helper.rb

```diff
@@ -52,14 +53,18 @@ def wait_for_deploy_to_finish(deployment, timeout = 600)
         if deployment['state'] == 'error'
           deployment['service_deploys'].each do |service_deploy|
             if service_deploy['state'] == 'error'
-              error service_deploy['reason']
+              errors << pastel.red("#{service_deploy['service_id']} deploy failed: #{service_deploy['reason']}")
```

Comment:
Just use `warn` or `$stderr.puts` to print out each of these errors, instead of using the `error` function that exits.